src/include/misc_inline.h

```diff
@@ -197,6 +197,36 @@ __wt_spin_backoff(uint64_t *yield_count, uint64_t *sleep_usecs)
     __wt_sleep(0, (*sleep_usecs));
 }
 
+/*
+ * __wt_timing_stress_sleep_for_secs --
+ *     Optionally add delay to stress code paths. Wait for the specified amount of time.
+ */
+static inline void
+__wt_timing_stress_sleep_for_secs(WT_SESSION_IMPL *session, uint32_t flag, struct timespec *tsp)
```

Comment:
The duplication from the other function doesn't seem ideal. Could you instead add a new flag to the `wt_timing_stress` function (something like "long wait"). The `__checkpoint_timing_stress` function also does something similar.

They should be folded together somehow.

Reply:
Yes, I agree with you. I will merge `__wt_timing_stress_sleep_for_secs ` and `__checkpoint_timing_stress ` with  `wt_timing_stress ` with some additional parameters.